superset/assets/visualizations/big_number.js

```diff
@@ -59,36 +68,41 @@ function bigNumberVis(slice, payload) {
 
   let y = height / 2;
   let g = svg.append('g');
+
+  const formattedNumber = f(v);
+
   // Printing big number
   g.append('g').attr('class', 'digits')
-  .attr('opacity', 1)
-  .append('text')
-  .attr('x', width / 2)
-  .attr('y', y)
-  .attr('class', 'big')
-  .attr('alignment-baseline', 'middle')
-  .attr('id', 'bigNumber')
-  .style('font-weight', 'bold')
-  .style('cursor', 'pointer')
-  .text(f(v))
-  .style('font-size', d3.min([height, width]) / 3.5)
-  .style('text-anchor', 'middle')
-  .attr('fill', 'black');
+   .attr('opacity', 1)
```

Comment:
no a biggy, but I'm surprised to see an odd indent. I'd go aligned or 2 spaces in for method chaining.